src/services/utils.ts

```diff
@@ -161,3 +161,18 @@ export const formatDateYYYYMMDD = (date: string) => {
 
   return `${year}-${month}-${day}`
 }
+
+export const checkIfStringsAreEmpty = (inputs: string[]) => {
```

Comment:
```suggestion
export const checkIfStringsAreEmpty = (inputs: string[]) => {
export const checkIfStringsAreEmpty = (strings: string[]) => {
```
You are checking strings right?

Reply:
Updated param name to strings